### ISSUE TITLE
first epochs val mAp when fine tune very low

### ISSUE THREAD

**Issue by nisrinaam29** [ultralytics/ultralytics#19413]
### Search before asking

- [x] I have searched the Ultralytics YOLO [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/orgs/ultralytics/discussions) and found no similar questions.

### Question

im finetuning yolov11n object detection on my custom dataset. The first epoch very low mAp is 0.154, is it normal? im not using COCO classes, use only my custom classes. 

### Additional

task: detect
mode: train
model: yolo11n.pt
data: /Users/sina/train_yolo/dataset.yaml
epochs: 100
time: null
patience: 100
batch: 8
imgsz: 640
save: true
save_period: -1
cache: false
device: cpu
workers: 1
project: null
name: train21
exist_ok: false
pretrained: true
optimizer: auto
verbose: true
seed: 0
deterministic: true
single_cls: false
rect: false
cos_lr: false
close_mosaic: 10
resume: false
amp: true
fraction: 1.0
profile: false
freeze: 10
multi_scale: false
overlap_mask: true
mask_ratio: 4
dropout: 0.0
val: true
split: val
save_json: false
save_hybrid: false
conf: null
iou: 0.7
max_det: 300
half: false
dnn: false
plots: true
source: null
vid_stride: 1
stream_buffer: false
visualize: false
augment: false
agnostic_nms: false
classes: null
retina_masks: false
embed: null
show: false
save_frames: false
save_txt: false
save_conf: false
save_crop: false
show_labels: true
show_conf: true
show_boxes: true
line_width: null
format: torchscript
keras: false
optimize: false
int8: false
dynamic: false
simplify: true
opset: null
workspace: null
nms: false
lr0: 0.01
lrf: 0.01
momentum: 0.937
weight_decay: 0.0005
warmup_epochs: 3.0
warmup_momentum: 0.8
warmup_bias_lr: 0.0
box: 7.5
cls: 0.5
dfl: 1.5
pose: 12.0
kobj: 1.0
nbs: 64
hsv_h: 0.015
hsv_s: 0.7
hsv_v: 0.4
degrees: 0.0
translate: 0.1
scale: 0.5
shear: 0.0
perspective: 0.0
flipud: 0.0
fliplr: 0.5
bgr: 0.0
mosaic: 0
mixup: 0.0
copy_paste: 0.0
copy_paste_mode: flip
auto_augment: randaugment
erasing: 0.4
crop_fraction: 1.0
cfg: null
tracker: botsort.yaml
save_dir: /Users/sina/runs/detect/train21

**Comment by UltralyticsAssistant** [comment]
👋 Hello @nisrinaam29, thank you for your interest in Ultralytics 🚀!

We recommend starting with the [Ultralytics Docs](https://docs.ultralytics.com/) for guidance on training and fine-tuning models. You can also explore our [Training Tips](https://docs.ultralytics.com/guides/model-training-tips/) and [YOLO Modes](https://docs.ultralytics.com/modes/) to get the most out of your training runs.

If this is a ❓ Question about custom training, early epochs might indeed show lower mAP values as the model begins to learn. To better assist with your issue, could you please try the following:
- Ensure your dataset setup aligns with the [Dataset Format Guide](https://docs.ultralytics.com/datasets/).
- Double-check that your `dataset.yaml` file is properly configured.
- Provide more details about your dataset and results, if possible, including example images, labels, and logs from training.

If this is a 🐛 Bug Report, please provide a [minimum reproducible example](https://docs.ultralytics.com/help/minimum-reproducible-example/) that demonstrates the issue so we can better debug it. This will help us to assist you more efficiently. 

If you’re running an older version of the `ultralytics` package, try installing the latest version to ensure the issue isn’t already resolved in a new release. You can upgrade with:
```bash
pip install -U ultralytics
```
Make sure you're running Python >=3.8 and PyTorch >=1.8. For verified environments, consider using one of the following setups:
- **Notebooks with free GPU**: <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"/></a> <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolo11"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **Cloud setups**: Follow our [Google Cloud](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/) or [AWS](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/) Quickstart Guides.
- **Docker setups**: Check out our [Docker Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/) <a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>.

Lastly, join the Ultralytics community where it suits you best! For real-time discussion, hop into [Discord](https://discord.com/invite/ultralytics) 🎧. For in-depth troubleshooting and collaboration, visit our [Discourse](https://community.ultralytics.com/). Or, share your questions on the [Ultralytics Subreddit](https://reddit.com/r/Ultralytics).

This is an automated response to help guide you. Rest assured, an Ultralytics engineer will review this issue and assist you further soon 😊.

## Status
<a href="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule"><img src="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml/badge.svg" alt="Ultralytics CI"></a> If this badge is green, all [Ultralytics CI](https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule) tests are currently passing. CI tests verify correct operation of all YOLO [Modes](https://docs.ultralytics.com/modes/) and [Tasks](https://docs.ultralytics.com/tasks/) across macOS, Windows, and Ubuntu every 24 hours and on every commit.

**Comment by Y-T-G** [comment]
It's not low. mAP is shown in fraction. Maximum is 1.0. 0.154 is 15.4 mAp.
